### ISSUE TITLE
Is there any optio to use this library over websocket

### ISSUE THREAD

**Issue by Elsaligi** [jeffbass/imagezmq#78]
Hi,

Is there any option to use this library over websocket. For enterprise network only 443 is open.
Please suggest?

**Comment by jeffbass** [comment]
No, there is no option to use imageZMQ with websockets. This is because websockets and ZMQ sockets are completely different communications / message protocols. They are not compatible.

There is a good answer on this [StackOverflow thread](https://stackoverflow.com/questions/26682634/differences-between-zeromq-and-websockets).

Quoting from one of the answers there: "WebSockets is just a protocol (similar to http), while ZeroMQ is protocol and server which is responsible for lifecycle of your message from the moment it is received until it's consumed."
